cmd/fleetctl/gitops_test.go

```diff
@@ -1810,11 +1820,371 @@ func setupFullGitOpsPremiumServer(t *testing.T) (*mock.Store, **fleet.AppConfig,
 	ds.UpdateVPPTokenFunc = func(ctx context.Context, tok *fleet.VPPTokenDB) error {
 		return nil
 	}
+	ds.ListABMTokensFunc = func(ctx context.Context) ([]*fleet.ABMToken, error) {
+		return []*fleet.ABMToken{{OrganizationName: "Fleet Device Management Inc."}}, nil
+	}
 
 	t.Setenv("FLEET_SERVER_URL", fleetServerURL)
 	t.Setenv("ORG_NAME", orgName)
 	t.Setenv("TEST_TEAM_NAME", teamName)
 	t.Setenv("APPLE_BM_DEFAULT_TEAM", teamName)
 
-	return ds, &savedAppConfig, &savedTeam
+	return ds, &savedAppConfig, savedTeams
+}
+
+func TestABMGitOps(t *testing.T) {
```

Comment:
in the follow up PR with the VPP changes, I'm improving this test to perform some assertions on the datastore method calls.